x-pack/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer_utils/get_focus_data.ts

```diff
@@ -30,11 +31,18 @@ export interface Interval {
   expression: string;
 }
 
+export interface ChartDataPoint {
+  date: Date;
+  value: number | null;
+  upper?: number | null;
+  lower?: number | null;
+}
+
 export interface FocusData {
-  focusChartData: any;
-  anomalyRecords: any;
+  focusChartData: ChartDataPoint[];
+  anomalyRecords: MlAnomalyRecordDoc[];
   scheduledEvents: any;
-  showForecastCheckbox?: any;
+  showForecastCheckbox?: boolean;
```

Comment:
Do you have forecast data showing up? I can't find a way of displaying data from a forecast in a chart. It might be best left for future work - it isn't needed for this first version, and you'd need to add a control into the initial modal for displaying which forecast to display.

Reply:
Forecasting will be added in a follow up - though I left some of the code that will be needed for it since we will need it anyway.